_docs/concepts/security/mutual-tls.md

```diff
@@ -86,15 +86,15 @@ For services running on Kubernetes pods, the per-cluster Istio Citadel (acting a
 
 * Revoke a specific key and certificate pair when necessary
 
-For services running on VM/bare-metal machines, the above four operations are performed by Istio Citadel together with node agents.
+For services running on VM/bare-metal machines, the above four operations are performed by Citadel together with node agents.
 
 ## Workflow
 
 The Istio Security workflow consists of two phases, deployment and runtime. For the deployment phase, we discuss the two scenarios (i.e., in Kubernetes and VM/bare-metal machines) separately since they are different. Once the key and certificate are deployed, the runtime phase is the same for the two scenarios. We briefly cover the workflow in this section.
 
 ### Deployment phase (Kubernetes Scenario)
 
-1. Istio Citadel watches Kubernetes API Server, creates a [SPIFFE](https://spiffe.github.io/docs/svid) key and certificate pair for each of the existing and new service accounts, and sends them to API Server.
+1. Citadel watches Kubernetes API Server, creates a [SPIFFE](https://spiffe.github.io/docs/svid) key and certificate pair for each of the existing and new service accounts, and sends them to API Server.
```

Comment:
watches the Kubernetes
them to the API server

Reply:
Done.